pyiron/atomistics/structure/pyscal.py

```diff
@@ -4,7 +4,7 @@
 
 import numpy as np
 from pyiron_base import Settings
-from pyiron.atomistics.structure.atoms import pyiron_to_ase
+import pyiron.atomistics.structure.atoms
```

Comment:
Why is this change required?

Reply:
Otherwise we end up in the circular dependencies:
In atoms we import analyze, in analyze we import pyscal, and if we also import  a function from atoms here, we close the cycle. https://github.com/pyiron/pyiron/pull/1240#discussion_r535205748 https://github.com/pyiron/pyiron/pull/1240#discussion_r535222360